src/components/screens/about-screen/about-screen.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+import { Dispatch } from "redux";
+import { connect } from "react-redux";
+import { NullableToken } from "../../../types";
+import strings from "../../../localization/strings";
+import { Container, Typography, WithStyles, withStyles } from "@material-ui/core";
+import AppLayout from "../../layouts/app-layout/app-layout";
+import { ReduxActions, ReduxState } from "../../../store";
+import styles from "../../../styles/screens/about-screen"
+
+
+/**
```

Comment:
```suggestion
/**
```
No need for two spaces here